src/main/kotlin/org/rust/ide/newProject/Utils.kt

```diff
@@ -25,10 +25,11 @@ fun Cargo.makeProject(
     project: Project,
     module: Module,
     baseDir: VirtualFile,
+    name: String,
     template: RsProjectTemplate
 ): GeneratedFilesHolder? = when (template) {
-    is RsGenericTemplate -> init(project, module, baseDir, template.isBinary)
-    is RsCustomTemplate -> generate(project, module, baseDir, template.url)
+    is RsGenericTemplate -> init(project, module, baseDir, template.isBinary, name)
+    is RsCustomTemplate -> generate(project, module, baseDir, name, template.url)
```

Comment:
Let's pass the arguments in the same order (`name` after `template...`)

Reply:
Maybe after `baseDir` then? To keep first 4 parameters the same in both functions. To make it look like this:
```
    init(project, module, baseDir, name, template.isBinary)
generate(project, module, baseDir, name, template.url)
```